configuration/etl/etl_sql.d/migrations/9.5.0-10.0.0/drop_schema_version_history.sql

```diff
@@ -0,0 +1,7 @@
+-- DROP the schema_version_history table in mod_hpcdb, mod_logger, moddb, modw
```

Comment:
```suggestion
-- DROP the schema_version_history table in mod_hpcdb, mod_logger, mod_shredder, moddb, modw
```

I'm against comments like this, but if you're going to include it then might as well include all the schemas.

Reply:
agreed about including all of the tables in the comment. What kind of comment would you prefer to see here?

Reply:
I don't think any comment is necessary.  If you really need one you could explain why the tables are being dropped.